.github/workflows/check-repository.yml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: "Run repolinter"
-        uses: simatic-ax/actions/lint-repository@stable
+        uses: simatic-ax/actions/lint-repository@test_repo_lint_fabian_unglert
```

Need go back to stable

```suggestion
        uses: simatic-ax/actions/lint-repository@stable
```